app/views/licences/tabs/set-up.njk

```diff
@@ -10,135 +10,207 @@
   {% endfor %}
 {% endmacro %}
 
-{% set chargeVersionRows = [] %}
-
-{% set chargeVersionHead = [
-  {
-    text: "Start date"
-  },
-  {
-    text: "End date"
-  },
-  {
-    text: "Reason"
-  },
-  {
-    text: "Status"
-  },
-  {
-    text: "Action"
-  }
-] %}
-
-{% for chargeVersion in  chargeInformation %}
-  {% set chargeVersionRows = (chargeVersionRows.push([
-    {
-      text: chargeVersion.startDate
-    },
-    {
-      text: chargeVersion.endDate
-    },
-    {
-      text: chargeVersion.reason
-    },
-    {
-      html: statusTag(chargeVersion.status)
-    },
-    {
-      html: createLink(chargeVersion)
-    }
-  ]), chargeVersionRows) %}
-{% endfor %}
-
 <h2 class="govuk-heading-l">Licence set up</h2>
 
+  <h3 class="govuk-heading-m"> Requirements for returns </h3>
+
+  {% if returnVersions.length == 0 %}
+    <p class="govuk-body"> No requirements for returns for this licence.</p>
+  {% endif %}
+
+  {% if returnVersions.length > 0 %}
+    {% set returnVersionsTableRows = [] %}
+
+    {% for returnsRequirement in returnVersions %}
+      {% set returnVersionsTableRows = (returnVersionsTableRows.push([
+        {
+          text: returnsRequirement.startDate
+        },
+        {
+          text: returnsRequirement.endDate
+        },
+        {
+          text: returnsRequirement.reason
+        },
+        {
+          html: statusTag(returnsRequirement.status)
+        },
+        {
+          html: createLink(returnsRequirement)
+        }
+      ]), returnVersionsTableRows) %}
+    {% endfor %}
+
+    {{ govukTable({
+        head: [
+        {
+          text: "Start date"
+        },
+        {
+          text: "End date"
+        },
+        {
+          text: "Reason"
+        },
+        {
+          text: "Status"
+        },
+        {
+          text: "Action"
+        }
+      ],
+        rows: returnVersionsTableRows
+      }) }}
+  {% endif %}
+{% if enableRequirementsForReturns %}
```

Comment:
Rather than have this check here, why not reference the feature flag in `SetUpPresenter`? You can then determine whether the link gets added to the page data there.

This simplifies the view even further and means you don't have to test for this using a controller test.